LDAR_Sim/src/scheduling/follow_up_mobile_schedule.py

```diff
@@ -106,3 +109,19 @@ def add_previous_queued_to_survey_queue(self, survey_plan: FollowUpSurveyPlanner
             (GenericSchedule.QUEUED_SURVEY_PRIORITY, survey_plan.rate_at_site),
             survey_plan,
         )
+
+    def update(self, workplan: Workplan, current_date: date) -> None:
```

Comment:
For now this is fine, but it's really only 1 line different from the supper - We either need a ticket or a note to look at cleaning this up so we can reuse existing code from the super

Reply:
This was better the other way even though it was more code